src/storage/query/QueryBaseProcessor-inl.h

```diff
@@ -152,6 +152,13 @@ nebula::cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::buildFilter(
     if (filter_ == nullptr) {
       return nebula::cpp2::ErrorCode::E_INVALID_FILTER;
     }
+    const auto* vertexFilterStr = getFilter(req, true);
+    if (vertexFilterStr != nullptr && !vertexFilterStr->empty()) {
+      vertexFilter_ = Expression::decode(pool, *vertexFilterStr);
```

Comment:
should `vertexFilter_` be check in `checkExp`?

Reply:
vertexfilter is a subset of filter_ and does not need to be checked repeatedly